cheroot/test/test_core.py

```diff
@@ -280,6 +286,17 @@ def commas(self):
 
         cherrypy.tree.mount(root)
 
+    @staticmethod
+    @contextlib.contextmanager
+    def temp_file(file, content):
```

Comment:
This method is completely detached from the class, I think it is better to turn it into a function in module level scope.